app/views/release_notes/_v5.0.2.1.html.erb

```diff
@@ -0,0 +1,17 @@
+ <h3>AACT 5.0.2.1<i>(October 28, 2021)</i></h3>
+ 
+ <h5>Release notes - AACT - Version aact-core-5.0.2</h5>
+ 
+ <h5>Bug Fixes</h5>
+ <ul>
+  <li><p>Add messages to data_verification to know what is happening</p></li>
+  <li><p>Fix Util::Updater tests</p></li>
+ </ul>
+
+ <h5>Features</h5>
+ <ul>
+  <li><p>As a user I want to be able to create flat files for bet</p></li>
+  <li><p>As a user I want to be able download the database snapshots for beta</p></li>
+  <li><p>As a developer I want an example connections.yml file</p></li>
+  <li><p>Write a script that tests single row/nct_id tables using SQL</p></li>
+ </ul>
```

Comment:
all these need to be reworded

- "Added messages to data_verification method to let you know what is happening"
- "Fixed Util::Updater tests"
- "Setup code to create flat files for beta"
- "Setup code to create snapshots for beta"
- "Added an example connections.yml file"
- "Added a script that compares between schemas tables with that only have one occurrence of each nct_id"